molo/core/tests/test_views.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+import pytest
+
+
+@pytest.mark.django_db
+class TestPages(TestCase):
+    fixtures = ['molo/core/tests/fixtures/test.json']
```

Comment:
Well, these tests being just for molo, it works (within this module). I doubt it will work when run in a subproject. I'll investigate when adding tests to `ndohyep`